### PR TITLE
[ADD] adding Minix name, my favorite and crazy cat.

### PR DESCRIPTION
**Pull request by waltervargas**
Hi @sindresorhus, Please accept this pull request, it has a special and emotional reason for several humans from Venezuela and Costa Rica :)

Regards.

### CODE REVIEW

File: cat-names.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
 	"Mia",
 	"Midnight",
 	"Milo",
+        "Minix",
```
Comment (kevva) [inline cat-names.json]: Could you fix indentation?

Comment (waltervargas) [inline cat-names.json]: Yes, let me check it.

**Comment by sindresorhus** [comment]
Sorry, but this list is locked as it's a list of **popular** cat names, and it doesn't make sense to arbitrary mutate it.

From the top of the readme:

> Not accepting PRs for additional names.

File: cat-names.json (hunk 1 above)
Comment (waltervargas) [inline cat-names.json]: @kevva Ready :)

**Comment by waltervargas** [comment]
@sindresorhus Sorry, my mistake by not read carefully

**Comment by sindresorhus** [comment]
@waltervargas I encourage you to fork this and make your own with random cat names though ;)

**Pull request closed without merging**